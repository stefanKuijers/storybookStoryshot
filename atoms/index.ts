export { default as Branding } from "./Branding/Branding";
export { default as Icon } from "./Icon/Icon";
export { default as Title } from "./Title/Title";
