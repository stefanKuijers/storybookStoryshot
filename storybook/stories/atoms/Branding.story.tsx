import { text } from "@storybook/addon-knobs";
import React from "react";

import { Branding } from "../../../atoms";
import { stageStoriesOf } from "../utils";

stageStoriesOf("Atoms", "Branding").add("Branding", () => (
  <Branding
    logo={text("Logo (src):", "https://i.ytimg.com/vi/4yikpWtIFU8/maxresdefault.jpg")}
    tagline={text("Tagline?:", "")}
  />
));
