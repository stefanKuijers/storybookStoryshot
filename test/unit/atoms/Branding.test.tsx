import React from "react";
import renderer from "react-test-renderer";

import { Branding } from "../../../atoms";

describe("Branding", () => {
  it("should match the snapshot", () => {
    const Renderer = renderer.create(
      <Branding
        logo={"https://i.ytimg.com/vi/4yikpWtIFU8/maxresdefault.jpg"}
        tagline={"Tag Line"}
      />
    );

    expect(Renderer.toJSON()).toMatchSnapshot();
  });
});
