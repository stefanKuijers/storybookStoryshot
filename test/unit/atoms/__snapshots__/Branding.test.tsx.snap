// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Branding should match the snapshot 1`] = `
<div
  className="Branding tclass-branding"
>
  <img
    className="Branding__logo"
    src="https://i.ytimg.com/vi/4yikpWtIFU8/maxresdefault.jpg"
  />
  <b
    className="Branding__text"
  >
    Tag Line
  </b>
</div>
`;
